### PR TITLE
jami-*: 20221031.1308.130cc26 -> 20221220.0956.79e1207

### PR DESCRIPTION
**Pull request by linsui**
###### Description of changes
Closes #207667

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/jami/client.nix [hunk 1]
```diff
@@ -31,14 +31,6 @@ stdenv.mkDerivation {
 
   sourceRoot = "source/client-qt";
 
-  patches = [
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/instant-messengers/jami/client.nix:35]: Please remove the unused fetchpatch from inputs.

Comment (linsui) [reply pkgs/applications/networking/instant-messengers/jami/client.nix:35]: Done.

**Review by panicgh — COMMENTED** [review]
`ffmpeg-jami` currently fails to build with "ERROR: sdl2 requested but not found": https://hydra.nixos.org/build/205624553/nixlog/1

This has been fixed in ffmpeg-5.0.2. Please consider updating it. You can cherry-pick it from https://github.com/panicgh/nixpkgs/commit/f4be429bd91745f99ce623d16c877f4846f2756a if you want.
(Diff between 5.0.1 and 5.0.2 with the fix: https://git.ffmpeg.org/gitweb/ffmpeg.git/commitdiff/2f428de9ebdcd0770de37d874871b25325aebd73?hp=9687cae2b468e09e35df4cea92cc2e6a0e6c93b3#patch4)

**Comment by linsui** [comment]
Not sure why but https://github.com/NixOS/nixpkgs/pull/204198 will fix it anyway.

**Comment by panicgh** [comment]
> Not sure why but #204198 will fix it anyway.

Thanks for pointing it out. It works because ffmpeg_5 is version 5.1.2 > 5.0.1 and already includes the fix for the SDL2 range check.

**Review by panicgh — APPROVED** [review]

**Comment by linsui** [comment]
@panicgh I fixed it anyway.

**Comment by linsui** [comment]
Thanks for merging!

**Pull request merged**